cea/optimization/slave/electricity_main.py

```diff
@@ -358,84 +358,80 @@ def extract_electricity_demand_buildings(master_to_slave_vars, building_names, l
     for name in building_names:  # adding the electricity demand of
         building_demand = pd.read_csv(locator.get_demand_results_file(name))
         # end-use electrical demands
-        Eal_req_W += building_demand['Eal_kWh'] * 1000
-        Edata_req_W += building_demand['Edata_kWh'] * 1000
-        Epro_req_W += building_demand['Epro_kWh'] * 1000
-        Eaux_req_W += building_demand['Eaux_kWh'] * 1000
+        Eal_req_W += (building_demand['Eal_kWh'] * 1000).values
+        Edata_req_W += (building_demand['Edata_kWh'] * 1000).values
+        Epro_req_W += (building_demand['Epro_kWh'] * 1000).values
+        Eaux_req_W += (building_demand['Eaux_kWh'] * 1000).values
 
     # when the two networks are present
     if master_to_slave_vars.DHN_exists and master_to_slave_vars.DCN_exists:
         for name in building_names:
             building_demand = pd.read_csv(locator.get_demand_results_file(name))
             if name in buildings_connected_to_district_heating and name in buildings_connected_to_district_cooling:
                 # if connected to the heating network
-                E_hs_ww_req_W += 0.0
-                E_cs_cre_cdata_req_W += 0.0
+                E_hs_ww_req_W += np.zeros(HOURS_IN_YEAR)
+                E_cs_cre_cdata_req_W += np.zeros(HOURS_IN_YEAR)
             elif name in buildings_connected_to_district_heating:
                 # if disconnected from the heating network
-                E_hs_ww_req_W += 0.0
+                E_hs_ww_req_W += np.zeros(HOURS_IN_YEAR)
```

Comment:
nice! while the effect is the same, this communicates the "arrayness" of the variable :)